android-lib/src/main/java/com/kinvey/android/store/DataStore.java

```diff
@@ -578,8 +578,8 @@ private void saveBatch(@NonNull List<T> entities, @NonNull KinveyClientCallback<
         Preconditions.checkArgument(client.isInitialize(), "client must be initialized.");
         Preconditions.checkNotNull(entities, "Entity cannot be null.");
         Preconditions.checkState(entities.size() > 0,"Entity list cannot be empty.");
-        Preconditions.checkPositionIndex(entities.size(), MAX_MULTI_INSERT_SIZE,
-             String.format(Locale.US, "Reached maximum of %d items per request.", MAX_MULTI_INSERT_SIZE));
+        //Preconditions.checkPositionIndex(entities.size(), MAX_MULTI_INSERT_SIZE,
+        //     String.format(Locale.US, "Reached maximum of %d items per request.", MAX_MULTI_INSERT_SIZE));
```

Comment:
I think better remove than comment in this case

Reply:
Fixed.